javascript-timers/interval.js

```diff
@@ -0,0 +1,18 @@
+var secondsElapsed = 0;
+var $h1 = document.querySelector('.countdown-display');
+
+setInterval(() => {
```

Comment:
the only update you need here is to clear the interval once the countdown is complete